docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+
+### 6. Adding a new variant in the middle of an experiment
+
+This is just a special case of #2.
+
+### 7. Removing a variant in the middle of an experiment
+
+This is just a special case of #2.
+
+### 8. Was sticky bucketing turned on and off in the middle of the experiment?
+
+Generally, once you start an experiment you should not change anything.
+
+### 9. Are there a lot of users in the ["Individual Users Allocation"](../general/evaluation/implementation.md#Individual-inclusions) section?
+
+This can cause an SRM because you are not randomly allocating those users. It is fine to add < 1% (// TODO Akhil do I want to be this prescriptive with a number) of the users in the experiment in the "Individual Users Allocation". 
+
+### 10. Is this your first experiment?
+
+Maybe there is an instrumentation bug. If you have had multiple SRMs, then it is unlikely to be a false positive.
+
+### 11. Are users logging in and out a lot?
+
+You may see this manifested in the variant jumping chart. Is the experiment causing users to log out more? This happens for a lot of financial institutions where you get logged out of your bank account if you are inactive for more than 15 minutes.<br>
+
+>    1. On log out regenerate device id<br>
+>    2. Amplitude thinks its a new user<br>
+>    3. User can get bucketed into a different variant<br>
+>    4. Then id gets resolved<br>
+
+This can also be related to anonymous users from sign up flow experiments. 
+
+### 12. Are there two users who have the same sequence of events but one is missing exposure?
+
+Maybe there are some cases where you are not sending the exposure event or are spending the exposure event when you should not be.
+
+### 13. If the customer is not using our sdk is the fall back variant causing issue?
+
+Is there some case where if the flag does not return a response in a reasonable amount of time, they default to control and count that as an exposure? Then control will have more exposures than treatment. 
+
+!!! info
+    Our client side sdk doesn’t send exposure events for fallback variant
+
+### 14. Look at the [Platform user property](https://help.amplitude.com/hc/en-us/articles/215562387-User-property-definitions)
+
+Are you sending data to Amplitude though [Segment](https://www.docs.developers.amplitude.com/data/sources/segment/#javascript-client-side), [Amplitude SDK](https://www.docs.developers.amplitude.com/analytics/what-is-amplitude/), [Batch API](https://www.docs.developers.amplitude.com/analytics/apis/batch-event-upload-api/), etc.<br>
+
+This is by no means an exhaustive list. Here is another debugging [strategy](https://www.lukasvermeer.nl/srm/docs/faq/#what-can-we-do-about-sample-ratio-mismatch). If you still are having issues debugging a SRM, please reach out to support or Customer Success by ... //TODO AKhil fill this in.
```

Comment:
@kathyqian Is there some preferred method for customers should reach out to support or Customer Success?

Reply:
No, I think it varies by customer